app/assets/stylesheets/pages/_howitworks.scss

```diff
@@ -1,3 +1,37 @@
+body {
```

Comment:
dont target the whole body or this will overwrite everything in our app. give classes to elements you want to style